courses/machine_learning/deepdive/05_review/6_deploy.ipynb

```diff
@@ -26,9 +26,13 @@
    "outputs": [],
```

Comment:
<p>output mentions "BrokenPipeError"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/730/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Yes. It happens when the bucket is already created. Not sure why, but it doesn't impact behavior. </p>